Exercise_2/index.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/style.css"/>
+</head>
+<body>
+ <header class="header">
+        <aside class="header__side-section">Social</aside>
+        <section class="header__central-section header__central-section_focus">City Guide</section>
```

Comment:
We played with BEM and forgot to remove `header__central-section_focus`. Please, remove it.